x/accounts/defaults/multisig/update_config.go

```diff
@@ -4,11 +4,17 @@ import (
 	"context"
 	"errors"
 
+	"cosmossdk.io/x/accounts/accountstd"
 	v1 "cosmossdk.io/x/accounts/defaults/multisig/v1"
 )
 
 // UpdateConfig updates the configuration of the multisig account.
 func (a Account) UpdateConfig(ctx context.Context, msg *v1.MsgUpdateConfig) (*v1.MsgUpdateConfigResponse, error) {
+	// this function can only be executed by the account itself
+	if !accountstd.SenderIsSelf(ctx) {
```

Comment:
It would be good to cover this check with tests as this is a key security feature. If it does not fit into the existing table tests, it can be new test.

Reply:
Added a test for this, thank you! The idea here is that only the multisig can update its own config, and the only way to achieve this is through an approved proposal.